main.js

```diff
@@ -0,0 +1,471 @@
+    }
+
+    while (
+      gameBoardArray[startY2] &&
+      gameBoardArray[startY2][startX2] === symbol
+    ) {
+      endCounter++;
+      startY2 += set["y2"];
+      startX2 += set["x2"];
+    }
+
+    if (startCounter > 2 && endCounter > 2) {
+      return true;
+    }
+  }
+  return false;
+}
+
+function checkForCrossPattern(y, x, symbol, vectors) {
+  var directions = [];
+
+  for (var direction in vectors) {
+    var startCounter = 1;
+    var endCounter = 1;
+    var set = vectors[direction];
+    var startY1 = y + set["y1"];
+    var startX1 = x + set["x1"];
+    var startY2 = y + set["y2"];
+    var startX2 = x + set["x2"];
+
+    while (
+      gameBoardArray[startY1] &&
+      gameBoardArray[startY1][startX1] === symbol
+    ) {
+      startCounter++;
+      startY1 += set["y1"];
+      startX1 += set["x1"];
+    }
+
+    while (
+      gameBoardArray[startY2] &&
+      gameBoardArray[startY2][startX2] === symbol
+    ) {
+      endCounter++;
+      startY2 += set["y2"];
+      startX2 += set["x2"];
+    }
+
+    if (startCounter > 1 && endCounter > 1) {
+      directions.push(true);
+    } else {
+      directions.push(false);
+    }
+  }
+  //check if both directions have matching symbols
+  if (directions[0] && directions[1]) {
+    return true;
+  }
+  return false;
+}
+
+function checkForWin(y, x, symbol) {
+  for (var majorDirection in vectors) {
+    var counter = 1;
+    var set = vectors[majorDirection];
+    var startY1 = y + set["y1"];
+    var startX1 = x + set["x1"];
+    var startY2 = y + set["y2"];
+    var startX2 = x + set["x2"];
+
+    //check to see if player1/ player2 or 0
+    //player1 was defined then an if statement to proceed
+    //only increment counter if it matches current player piece
+    // we need to check if on same axis, if not on same axis and the symbol isnt the same
+    //we switch to a new axis
+    while (
+      gameBoardArray[startY1] &&
+      gameBoardArray[startY1][startX1] === symbol
+    ) {
+      counter++;
+      startY1 += set["y1"];
+      startX1 += set["x1"];
+    }
+
+    while (
+      gameBoardArray[startY2] &&
+      gameBoardArray[startY2][startX2] === symbol
+    ) {
+      counter++;
+      startY2 += set["y2"];
+      startX2 += set["x2"];
+    }
+
+    if (counter >= 4) {
+      return true;
+    }
+  }
+  return false;
+}
+
+//this upon reset
+function displayModal() {
+  $("#modalShadow").show();
+  // document.querySelector("#modalShadow").style.display = "block";
+}
+
+function closeModalatStart() {
+  $("#modalShadow").hide();
+  createCells(7, 7);
+  createPlayerStats();
+  glowSelectedPlayerProfile();
+  createResetButton();
+  // toggleWinModalVisibility();
+}
+
+function resetGame() {
+  console.log("reset clicked");
+  gameBoardArray = [
+    [0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0]
+  ];
+  player = 0;
```

Comment:
You reset the player and change your board color however you don't reset the player goal. This makes it unclear as to who's turn it is. 
<img width="947" alt="screen shot 2018-05-30 at 11 59 38 am" src="https://user-images.githubusercontent.com/8095841/40741760-1ee83df8-6401-11e8-9c45-5992b3b9b2d6.png">